workspaces/bulk-import/plugins/bulk-import-backend/src/generated/openapi.d.ts

```diff
@@ -284,15 +284,14 @@ declare namespace Components {
       github?: {
         pullRequest?: PullRequest;
       };
-      source?: /**
+      source /**
        * Import Source:
        *   * 'config' - Import from static catalog location configuration in 'app-config'
        *   * 'location' - Import of user registered entities using locations endpoint
        *   * 'integration' - Import using a GitHub integration
        *   * null - Import source is unknown
        *
-       */
-      Source;
+       */?: Source;
```

Comment:
This change looks wired. Can you maybe exclude `generated` from prettier in the bulk import and run the generate api step again?

Reply:
I re-generated the api using this command `yarn workspace @red-hat-developer-hub/backstage-plugin-bulk-import-backend run openapi` but it resulted in a number of changes.  LMK if this is expected